### PR TITLE
chore(readme): Add KSM Address for Donations

### PR DESCRIPTION
**Pull request by ansermino**
## Changes

Adds KSM address for donations. Reformat donations section.

## Tests

N/A

## Issues

N/A 

## Primary Reviewer

@timwu20 @danforbes

### CODE REVIEW

**Review by jimjbrettj — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -162,9 +162,10 @@ To start the polkadot node:
 
 ## Donate
 
-Our work on Gossamer is funded by the community. If you'd like to donate, you can send DOT to
-[`14gaKBxYkbBh2SKGtRDdhuhtyGAs5XLh55bE5x4cDi5CmL75`](https://polkadot.subscan.io/account/14gaKBxYkbBh2SKGtRDdhuhtyGAs5XLh55bE5x4cDi5CmL75);
-ETH or DAI can be sent to `0x764001D60E69f0C3D0b41B0588866cFaE796972c`.
+Our work on Gossamer is funded by the community. If you'd like to support us with a donation:
+- DOT: [`14gaKBxYkbBh2SKGtRDdhuhtyGAs5XLh55bE5x4cDi5CmL75`](https://polkadot.subscan.io/account/14gaKBxYkbBh2SKGtRDdhuhtyGAs5XLh55bE5x4cDi5CmL75)
+- KSM: `FAjhFSFoM6X8CxeSp6JE2fPECauCA5NxyB1rAGNSkrVaMtf`
```
Comment (qdm12) [inline README.md:167]: Would there be a link to it like for the DOT? That would be fun for users to check it out

**Review by qdm12 — APPROVED** [review]

**Comment by ansermino** [comment]
@danforbes Unfortunately the address doesn't appear on Subscan as it has made no txs. We don't currently hold any KSM either 🤷 

Please don't wait on me to merge.

File: README.md (hunk 1 above)
Comment (ansermino) [reply README.md:167]: Good question. See PR comment.

**Comment by jimjbrettj** [comment]
I sent a small amount! Maybe we will have link now?

**Comment by jimjbrettj** [comment]
@ansermino @qdm12 does this work now? https://explorer.polkascan.io/kusama/account/FAjhFSFoM6X8CxeSp6JE2fPECauCA5NxyB1rAGNSkrVaMtf

**Comment by ansermino** [comment]
@jimjbrettj Saving the day! Updated. Thank youuuuuu <3

**Pull request merged**